src/NuGet.Core/NuGet.Protocol/Plugins/InboundRequestContext.cs

```diff
@@ -37,7 +38,7 @@ public sealed class InboundRequestContext : IDisposable
             IConnection connection,
             string requestId,
             CancellationToken cancellationToken)
-            : this(connection, requestId, cancellationToken, PluginLogger.DefaultInstance)
+            : this(connection, requestId, cancellationToken, new InboundRequestProcessingHandler(), PluginLogger.DefaultInstance)
```

Comment:
How about a `InboundRequestProcessingHandler.Default` instance?

Reply:
Added that and removed the default constructor.

Reply:
Decided to go against this change. 
The disposable pattern and default instances do not play well together :)
All of my tests failed :)